include/dev/wddr/table_vals.h

```diff
@@ -688,11 +688,11 @@
     .dfi.paden_pext.rd.fields.ren = 0x0,                                                                               \
     .dfi.ovr_traffic.sel_dq_wr = 0x0,                                                                                  \
     .dfi.ovr_traffic.sel_dqs = 0x0,                                                                                    \
-    .dfi.ovr_traffic.sel_ca = 0x0,                                                                                     \
+    .dfi.ovr_traffic.sel_ca = 0x1,                                                                                     \
```

Comment:
Should these get updated in the DDR_TABLES.xlsx file as well?  Are we still tracking that file in the repo?

Reply:
file isn't tracked in the repository. But it has been updated in Sharepoint.

Reply:
We decided a while ago that the XLSX shouldn't be public.